src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -2655,7 +2655,7 @@ public ConstructorEdit(INamedTypeSymbol oldType)
                                         // Associated member declarations must be in the same document as the symbol, so we don't need to resolve their symbol.
                                         // In some cases the symbol even can't be resolved unambiguously. Consider e.g. resolving a method with its parameter deleted -
                                         // we wouldn't know which overload to resolve to.
-                                        if (TryGetAssociatedMemberDeclaration(oldDeclaration, out var oldAssociatedMemberDeclaration))
+                                        if (TryGetAssociatedMemberDeclaration(oldDeclaration, EditKind.Delete, out var oldAssociatedMemberDeclaration))
```

Comment:
>EditKind [](http://example.com/codeflow?start=94&length=8)

I'm not sure I understand why this is needed. When a property is deleted we want to create semantic edits for all of its accessors, but not for the property itself. If an accessor is deleted we want a single semantic edit just for that accessor. Correct?

If the entire property is deleted wouldn't we want to skip creating deletes for its accessors represented by the deletes of their syntax nodes, since we will add deletes for all of the accessor for the deleted property node?

Reply:
I _think_ this was just for ease of control flow, so that a delete of a single accessor didn't end up in passing this if, failing the inner one, and then not being considered as a method symbol delete. If I take your other suggestion about extracting the below to a method, I should be able to call that new method here and revert this change.

Reply:
Though, that would mean this branch would look identical to the next branch, but I can't combine them without effectively bringing this change back 🤷‍♂️

Reply:
Yeah, don't think I can do anything meaningful about this... The problem is type parameters, which would fail the `newContainingType == null` check below. So to remove this, means adding a "is type parameter" check around that, which doesn't seem like it actually saves anything.

Reply:
@tmat how strongly do you feel about this? The other feedback has been addressed.